### PR TITLE
bug/ads-804: Fieldset & form spacing

### PR DESCRIPTION
**Pull request by cblva**

### CODE REVIEW

File: public/styles/components/_form.scss [hunk 1]
```diff
@@ -48,6 +48,12 @@
   padding-left: 0;
   padding-inline: 0;
   padding-block: 0;
+
+  &:first-child {
+    .bsds-form-control {
+      margin-top: space(2x);
```
Comment (kaseyllabelle) [inline public/styles/components/_form.scss:54]: add
```
      & [class='bsds-fieldset'] {
        margin-top: space(4x);
      }
```

File: src/components/Form/Form.stories.tsx [hunk 2]
```diff
@@ -36,46 +35,28 @@ export const Playground: Story = {
...
+        <InputText label="Required text input" required />
+        <InputText label="Disabled text input" disabled />
+        <Textarea label="Textarea" />
+        <Select label="Select">
+          <Option value="" disabled selected />
+          <Option value="option1">Option 1</Option>
+          <Option value="option2">Option 2</Option>
+          <Option value="option3">Option 3</Option>
+        </Select>
+        <Fieldset legend="Checkbox group" helpText={helpText} helpId="groupWithHelpId" className="bsds-mt-4x">
```
Comment (kaseyllabelle) [inline src/components/Form/Form.stories.tsx:48]: remove className in lieu of fieldset styles in form.scss

File: src/components/Form/Form.stories.tsx [hunk 3]
```diff
@@ -36,46 +35,28 @@ export const Playground: Story = {
...
+          <InputCheckbox label="Checkbox 1" />
+          <InputCheckbox label="Checkbox 2" />
+          <InputCheckbox label="Checkbox 3" />
+        </Fieldset>
+        <RadioGroup legend="Radio group" className="bsds-mt-4x">
```
Comment (kaseyllabelle) [inline src/components/Form/Form.stories.tsx:53]: remove className

File: src/components/Fieldset/Fieldset.tsx [hunk 4]
```diff
@@ -49,7 +49,10 @@ const Fieldset = ({
           {helpText}
         </p>
       )}
-      {children}
+
```
Comment (kaseyllabelle) [inline src/components/Fieldset/Fieldset.tsx:52]: delete whitespace and drop child onto new line

File: src/components/Form/Form.tsx [hunk 5]
```diff
@@ -15,7 +15,9 @@ const Form = ({ children, onInvalid, className, ...formAttrs }: FormProps): JSX.
 
   return (
     <form className={`bsds-form ${className || ''}`} onInvalid={handleInvalid} {...formAttrs}>
-      {children}
+      {React.Children.map(children, (child) => (
+        <div className="bsds-form-control">{child}</div>
```
Comment (kaseyllabelle) [inline src/components/Form/Form.tsx:19]: drop child

Comment (cblva) [reply src/components/Form/Form.tsx:19]: Linters won't let me drop the children in both files 😢

File: public/styles/components/_form.scss (hunk 1 above)
Comment (cblva) [reply public/styles/components/_form.scss:54]: This exact snippet wasn't working as expected, but I added a margin to fieldset within form on line 115

**Review by APJohns — APPROVED** [review]

**Pull request merged**